packages/aft/pubspec.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
   aws_common: ^0.1.0
   checked_yaml: ^2.0.0
   cli_util: ^0.3.5
+  interact: ^2.1.1
```

Comment:
This has a non-negligible impact on the security profile of this package. Are we prepared to manage that and are there better alternatives (like implementing a CLI selector, or pinning to a specific version)?

Reply:
Pinned the versions for this and `very_good_test_runner`.